addons/mail/models/mail_channel.py

```diff
@@ -836,6 +836,7 @@ def channel_info(self, extra_info=False):
                 partner_channel = channel_partners.filtered(lambda pc: pc.partner_id.id == self.env.user.partner_id.id)
                 if partner_channel:
                     partner_channel = partner_channel[0]
+                    info['members'] = partner_channel.mail_channel_partner_format(partner_infos)
                     info['state'] = partner_channel.fold_state or 'open'
```

Comment:
All of those existing keys of `partner_channel` need to be merged into the result of `mail_channel_partner_format()`. They are values on the same model, so they should be at the same level/in the same dict in the result.

Reply:
I don't remember if the point was settled, but as we discussed earlier, I think it's better to have those values unambiguously sent once at the thread level instead of as fields of members, since from the client point of view those fields only exist once per thread and not once per member. Although latter values like `is_mute` or `is_live` will definitely come from `mail_channel_partner_format()`.